src/frontend/components/isbn-registry/subComponents/modals/PublicationRequestCreationModal.jsx

```diff
@@ -95,32 +95,30 @@ function PublicationRequestCreationModal(props) {
 
   // Handles approving a process of adding a new ISBN/ISMN publication request
   const handleApproveAddingRequest = async () => {
-    setOpenModal(false);
-
     const formattedValues = {
       // Format the values to match the API requirements
       officialName: form.publisherName,
       title: form.publicationName,
       subtitle: form.publicationSubTitle || '',
       firstName1: form.authorName.split(' ').shift(),
       lastName1: form.authorName.split(' ').pop(),
-      email: form.contactEmail || '',
+      email: form.contactEmail,
       publicationFormat: form.publicationType,
-      type: [form.printFormat],
-      address: form.publicationAddress || '',
-      fileformat: [form.fileFormat],
+      type: form.printFormat,
+      address: form.publicationAddress,
+      fileformat: form.fileFormat,
       city: form.placeOfPublication,
       year: form.publicationYear.toString(),
       month: moment().month(form.publicationMonth).format('MM'),
-      // Set other required fields to placeholder values
+      // Set other required fields to placeholder values, since some sort of value is required by the API (can not be completely empty)
       role1: ['AUTHOR'],
       zip: '00000',
```

Comment:
We cannot have placeholder values similar to this. API needs changes for administrator validation so that empty strings are accepted were they can be accepted in business logic.

As a side note: In future we should aim for the database schema to reflect the real world use case and try to use null values when there is no value to be saved.